<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReosAddHydrographNodeFromWidget</class>
 <widget class="QWidget" name="ReosAddHydrographNodeFromWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Hydrograph Node From...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mProviderComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mProviderLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mCloseButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mAddNodeCopy">
       <property name="toolTip">
        <string>A node will be created containing a copy of this hydrograph that will be save on the project file.</string>
       </property>
       <property name="text">
        <string>Add  Node From Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mAddNode">
       <property name="toolTip">
        <string>A node will be created containing this hydrograph which will remain linked to the source.</string>
       </property>
       <property name="text">
        <string>Add Node</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
